### DISCUSSION TITLE
is enableEditing key in the columnDef could access row & column?

### DISCUSSION THREAD

**Discussion by AdityaBhadange** [KevinVandy/material-react-table#289] [Ideas]
is enableEditing key in the columnDef could access row & column? Because I want to conditionally enable/disable the editing the particular column.

I'm imagining something like this -
```
...other column defs
enableEditing: ({ column, row }) => {
  if (row.original.someKey === 'something') {
    return true;
  }
  return false
}
```

**Comment by KevinVandy** [comment]
duplicate of https://github.com/KevinVandy/material-react-table/discussions/253

See above on how to disable the textfields conditionally

**Reply by AdityaBhadange** [reply]
Thanks. I did the same!

**Comment by KevinVandy** [comment]
@AdityaBhadange  MRT v1.8.0 now let's you enable editing per row conditionally

```js
enableEditing: (row) => row.original.someCondition
```

**Reply by AdityaBhadange** [reply]
Thanks alot!
